extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -14,24 +14,16 @@
  * limitations under the License.
  */
 
-import {isJsonScriptTag, childElementsByTag} from '../../../src/dom';
+import {CSS} from '../../../build/amp-viz-vega-0.1.css';
+import * as dom from '../../../src/dom';
```

Comment:
@erwinmombay: How does DCE handle this? We only need 3 methods, hopefully the entire module isn't being imported.